### PR TITLE
Add publications section, services and update the team.

### PR DESCRIPTION
**Pull request by GabrielBrascher**
This PR aims to update the services and team sections. Also, it adds a publications section.

### CODE REVIEW

**Comment by GabrielBrascher** [comment]
Any help is welcome :+1: 
ping @rafaelweingartner @cristofolini @pedro-martins @alexandrelimassantana

**Comment by rafaelweingartner** [comment]
@GabrielBrascher it is great. 
What about adding a link around the "be part of our history" point to "https://gust.com/companies/autonomiccs". It is an open page, you do not need to be logged in to access it.

Also, what about using the same standard at the second publication? I mean, the contracted authors name and the quotes around the name of the publication.

File: index.html [hunk 1]
```diff
@@ -158,9 +161,14 @@ <h3 class="section-above-subheading text-muted">The Autonomiccs project is <a hr
                 <div class="col-lg-12 text-center">
                     <h2 class="section-heading">Services</h2>
                     <h3 class="section-above-subheading text-muted"></h3>
-                    <h3 class="section-subheading text-muted"></h3>
+                    <h3 class="section-subheading text-muted"></h3> 
                 </div>
             </div>
+            <div class="col-lg-12 text-center">
+                    <h3 class="section-heading">Cloud Management as a Service (CMaaS)</h3>
+                    We does not provide a product, but cloud computing autonomic management solutions.<br>
```
Comment (cristofolini) [inline index.html]: "We **do** not provide..." ;)

File: index.html [hunk 2]
```diff
@@ -204,6 +212,53 @@ <h3 class="section-subheading text-muted"></h3>
...
+                        <div class="portfolio-hover">
+                            <div class="portfolio-hover-content">
+                                <i class="fa fa-plus fa-3x"></i>
+                            </div>
+                        </div>
+                        <img src="img/portfolio/support.png" class="img-responsive" alt="">
+                    </a>
+                    <div class="portfolio-caption">
+                        <h4>Consulting and Support</h4>
+                        <p class="text-muted">Have our support for the management of your cloud. Enjoy a team ready for help with unpleasant issues.</p>
```
Comment (cristofolini) [inline index.html]: "Ready **to** help"

**Comment by cristofolini** [comment]
@GabrielBrascher Nice work! Other than the two small corrections, it seems Alexandre's bio has my e-mail and mine has Pedro's.

**Comment by GabrielBrascher** [comment]
Some copy paste fail there, will correct it. Thanks for the review @cristofolini .

**Comment by GabrielBrascher** [comment]
@rafaelweingartner now there is a link at the "be part of our history!". But probably many will not find it.
I will think on a more intuitive way to present the link (for example expand the div when mouseover).

**Comment by rafaelweingartner** [comment]
Well, when the mouse is over that symbol. Is it a pointer normally, right? When we embedded an image into a link, the mouse pointer transforms into a handish symbol; web users are normally used to click on anything in which that icon appears.

**Comment by GabrielBrascher** [comment]
Recently I worked with a website that an image changed its color when the mouse was over it (and as expected the mouse itself changes). But almost everyone that I presented didn't clicked there.

My concern is with the "interest" on putting the mouse over the image/link, with the intention to discover if it has a link. Maybe a  mouseover with a bigger div, or by just scrolling the page it gets some indicative that someone can click there. Any indicative that it would be clickable.

I am trying to expand the designer and usability part of me ;)

**Comment by rafaelweingartner** [comment]
Got it, so let's see if we can figure out some ways to improve it

**Comment by GabrielBrascher** [comment]
Hey guys.
Updated this PR following our latest discussions of "Cloud Management aaS".

**Comment by GabrielBrascher** [comment]
@cristofolini @pedro-martins @alexandrelimassantana updated after @rafaelweingartner review.

**Comment by cristofolini** [comment]
Line 101: "...virtual machine scheduler software **that components** to manage..." - I think something's missing over here :)

Line 122: "**Make** sure that..." - That's supposed to be _Makes_, "Make" sounds imperative;

Line 155: "...**What** about using our..." - _How_ about...

Line 272: "...can customize ~~the~~ Apache CloudStack..."

Line 316: "...that helped us to **understand better** the cloud..." - to _better understand_

Line 553: "...You can **let** that exhausting job **for** our agents;..." - _leave_ [...] _to_ [...] ; "...do not have **swing moods**..." - _mood swings_

Line 556: "...that shows **physical VMs placement**..." - _placement of physical VMs_

Line 577: "...After **logged** at our web platform..." - _logging in_

Line 584: "...allow balancing, **power** on/off idle hosts..." - _powering_

Line 591: "...Therefore our 24/7 support does not guarantee that your call will be answered; it also assures that trouble-shooting can continue 24/7." - Does it or does it not? 

Line 605: "...personalized solution that **focus** on..." - _focuses_

Line 612: "Have a cloud expert of our team arriving at your site to solve problems, minimize your losses and get your Cloud on tracks again. With this service, you can sleep safe and sound at night." - Are we subject to any time constraints?

Line 790: "...new features that **delivers** ~~the~~ management as **you need**..." - _deliver_ ; _needed_

That's it as far as the text itself is concerned...

I think this gives a pretty good introduction to the CMaaS idea, but since it is kind of a novel concept at this point, we should probably have a link somewhere around line 556 prompting the reader to send us any questions they may have about the model... Something like a mailto:sales@autonomiccs.com.br link saying "Ask us any questions!"... Considering that, at the bottom line, we are offering a service, being communicative and responsive might be an interesting differential :)

**Comment by GabrielBrascher** [comment]
Thanks @cristofolini , good review as always.

What about we remove **physical** from line 556? I think that it is one redundancy that may cause more confusion than help.
Line 556: "...that shows **physical placement of** VMs ..." - _placement of VMs_

**Comment by GabrielBrascher** [comment]
About the time constraints, we need to think better about it.
I imagine on setting different constraints for on-site support, given where the client is located etc. Probably Platinum will have an SLA contract for each client.

**Comment by cristofolini** [comment]
@GabrielBrascher Yeah, that's actually a good idea to remove _physical_ in that context.

About, the time constraints, I agree with what you said. We can probably define them as they come...

**Pull request merged**